packs/factions.db

```diff
@@ -3,7 +3,7 @@
+{"_id":"7MBSH5duBbixRcdi","name":"The Crows","permission":{"default":0,"5izWgADOTEOz6xAL":3},"type":"faction","data":{"description":"An old gang with new leadership. Known for running illegal games of chance and extortion rackets.","type":"Underworld","tier":"2","goal_1":"Reestablish control of Crow’s Foot","goal_1_clock_max":6,"goal_2":"Rise in Tier","goal_2_clock_max":6,"turf":"Claims all of Crow’s Foot as their turf. Everyone in the district pays up the chain to them. HQ in an abandoned City Watch tower. Operates gambling dens in Crow’s Foot and extortion rackets at the Docks.","assets":"A veteran gang of thugs and killers. A number of small boats. A fortified HQ.","quirks":"Roric’s body was lost during his murder (it fell into a canal). His vengeful ghost is now at large in the city.","notables":"Lyssa (leader, brash, killer, noble family). Bell (second-in-command, loyal).","allies":"The Bluecoats, Sailors, The Lost, Citizens of Crow’s Foot.","enemies":"The Hive, Inspectors, Dockers.","situation":"Lyssa murdered the former boss of the Crows, Roric. She is a fearsome killer, and few want to cross her, but her position as leader of the Crows is uncertain. Some were very loyal to Roric. As the power-play continues, the Crows’ hold on the district just might slip away.","goal_clock":0,"hold":{"value":1,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icon.4_27.png","notes":"","effects":[]}u
```

Comment:
Looks like there is a typo. Letter "u" is added to the end of that string.

Reply:
Ah, weird. I can address tomorrow.

Reply:
Removed.